### DISCUSSION TITLE
Invalid attribute name.glTF Validator(MESH_PRIMITIVE_INVALID_ATTRIBUTE)

### DISCUSSION THREAD

**Discussion by HM1-1AM** [zeux/meshoptimizer#730] [Q&A]
Hi,

   I am using .gltf file to transfer scene information to unreal engine 5. 

When I try to add additional information to .gltf file such as "pshape": 2,"radius3": 3 information (see screenshot below), there is an error "Invalid attribute name.glTF Validator(MESH_PRIMITIVE_INVALID_ATTRIBUTE)". 

So my question is, is this the right way to add or there is something else I need to do to add this info to the .gltf file.

![Capture](https://github.com/user-attachments/assets/b5b39780-5a32-4b13-b280-434c359d6d31)

**Comment by donmccurdy** [comment]
@HM1-1AM perhaps you meant to file this discussion on the glTF Validator repository (https://github.com/KhronosGroup/glTF-Validator)? Valid attribute names are listed in the [glTF spec](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#meshes-overview), custom attributes must be prefixed by an underscore. For example, `_PSHAPE` and `_RADIUS3` should pass validation.
